Core/include/Acts/Seeding/Hashing/HashingAnnoy.ipp

```diff
@@ -0,0 +1,147 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2024 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "Acts/Definitions/Algebra.hpp"
+#include "Acts/Definitions/Units.hpp"
+
+#include <map>
+#include <set>
+#include <vector>
+
+bool LayerSelection(double r2, double z) {
```

Comment:
- Can this be inside an anonymous namespace
- We use `camelCase` for function names (same below)

Reply:
Right, I overlooked that this is included as part of the header

Reply:
I used `Acts::Details::HashingGeometry` for the name of the namespace, as suggested. 
It seems to not have any issue about being unused anymore, thanks.

Reply:
Sorry, the convention is actually `Acts::detail` like here https://github.com/acts-project/acts/blob/e4a3ec90a5b4991923485a58dbb25f39abce1c15/Core/include/Acts/Propagator/detail/Auctioneer.hpp#L14

Reply:
I updated it, thanks!